.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -262,15 +247,18 @@ runs:
         aws cloudformation describe-stack-events --stack-name $stack_name
       done
   - name: install prometheus
+    if: ${{ !inputs.private_cluster }}
     uses: ./.github/actions/e2e/install-prometheus
     with:
       account_id: ${{ inputs.account_id }}
       role: ${{ inputs.role }}
-      region: ${{ inputs.prometheus_region }}
+      prometheus_region: ${{ inputs.prometheus_region }}
+      region: ${{ inputs.region }}
```

Comment:
Are you thinking of making an update here?

Reply:
Nope I am not, I guess we discussed this yesterday why these two are different. It is because the prometheus could be different from the region from where the tests are running and this new variable is being used to point to the region from where pull through cache will work.

Reply:
I'm in complete alignment on with on having it here. I was hoping we could update `region` to be a little bit more descriptive

Reply:
Nvm, I'm agree, we should not change `region`